Marble Blast Platinum/platinum/client/scripts/optionsGui.cs

```diff
@@ -507,12 +507,15 @@
 $Options::Name    ["Online", 6] = "chatMessageSize";
 $Options::Title   ["Online", 6] = "In-Game Lines of Chat";
 $Options::Type    ["Online", 6] = "value";
-$Options::Name    ["Online", 7] = "autoLogin";
-$Options::Title   ["Online", 7] = "Auto Login";
+$Options::Name    ["Online", 7] = "disableTaunts";
+$Options::Title   ["Online", 7] = "Disable Taunts";
```

Comment:
Maybe it should be called "Disable Taunt Sounds" to be slightly more clear. Not very important tho

Reply:
![image](https://user-images.githubusercontent.com/7307599/231637108-d2dfd292-4fc2-4ecf-84a8-c442da21cea7.png)

Ok lol it's currently very confusing in the menu. It says "Disable Taunts" and "Disabled" means that the taunts are _not_ disabled. Probably "Mute Taunts" would be the best

Reply:
Def sounds better